src/index.js

```diff
@@ -38,26 +41,39 @@ export class State {
   }
 
   cursor() {
-    return cursor(this.data, [], this.replace.bind(this))
+    return cursor(this.data, [], this.replace.bind(this), this.getLastChanged.bind(this))
   }
 
-  replace(newData) {
+  replace(newData, keyPath) {
+    this.lastChangedKeyPath = keyPath
     this.data = newData
     this._events.emit('update', newData)
   }
+
+  getLastChanged() {
+    return this.lastChangedKeyPath
+  }
 }
 
 export class Cursor {
 
-  constructor(data, keyPath, replace) {
+  constructor(data, keyPath, replace, getLastChanged) {
 
     if (!data) throw new Error("Cursor missing data")
     if (!keyPath) throw new Error("Cursor missing keyPath")
     if (!replace) throw new Error("Cursor missing replace")
+    if (!getLastChanged) throw new Error("Cursor missing getLastChanged")
 
     this._data = data
     this._keyPath = keyPath
     this._replace = replace
+    this._getLastChanged = getLastChanged
+  }
+
+  changedInLastUpdate() {
+    var myPath = this._keyPath.join('.')
+    var changedPath = this._getLastChanged().join('.')
+    return changedPath.indexOf(myPath) == 0 || myPath.indexOf(changedPath) == 0
```

Comment:
I think this `indexOf` check might be a little brittle. What if the changedPath is `['foo', 'bar', 'baz']` and `myPath` is `['foo', 'bar', 'bazbeans']`? It will return true but it won't actually have changed.